osquery/tables/applications/posix/prometheus_metrics.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <sstream>
+
+#include <osquery/config.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include <osquery/core/conversions.h>
+
+#include <osquery/config/parsers/prometheus_targets.h>
+#include <osquery/tables/applications/posix/prometheus_metrics.h>
+
+namespace http = boost::network::http;
+
+namespace osquery {
+namespace tables {
+
+void parseScrapeResults(const std::map<std::string, retData>& scrapeResults,
+                        QueryData& rows) {
+  for (auto const& target : scrapeResults) {
+    std::stringstream ss(target.second.content);
+    std::string dest;
+    std::string ts(std::to_string(target.second.timestampMS.count()));
```

Comment:
As you're only using this to populate the row below, let's reduce it's scope and remove this line. Then down below you could have:
`r[kColTimeStamp] = BIGINT(target.second.timestampMS.count())`